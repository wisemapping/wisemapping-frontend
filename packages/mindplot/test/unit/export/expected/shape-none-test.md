# Central Topic

- Topic with undefined shape (uses theme default)
- Topic with explicit none shape
- Topic with line shape
- Topic with rectangle shape
