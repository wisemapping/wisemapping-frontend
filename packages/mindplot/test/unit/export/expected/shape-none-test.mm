<map version="1.0.1">
    <node ID="ID_1" TEXT="Central Topic">
        <node ID="ID_2" POSITION="right" STYLE="fork" TEXT="Topic with undefined shape (uses theme default)"/>
        <node ID="ID_3" POSITION="right" STYLE="none" TEXT="Topic with explicit none shape"/>
        <node ID="ID_4" POSITION="right" TEXT="Topic with line shape"/>
        <node ID="ID_5" POSITION="right" STYLE="rectangle" TEXT="Topic with rectangle shape"/>
    </node>
</map>