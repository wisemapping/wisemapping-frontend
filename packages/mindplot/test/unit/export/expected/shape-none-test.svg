<svg xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMinYMin" width="582" height="402" viewBox="-110 -130 581.5  402" style="background-color:white">
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C53.77,0 107.53,-100.00 161.30,-100.00 107.53,-97.00 53.77,5.00 0.00,7.00 Z"/>
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C53.77,0 107.53,0.00 161.30,0.00 107.53,3.00 53.77,5.00 0.00,7.00 Z"/>
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C53.77,0 107.53,100.00 161.30,100.00 107.53,103.00 53.77,5.00 0.00,7.00 Z"/>
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C53.77,0 107.53,200.00 161.30,200.00 107.53,203.00 53.77,5.00 0.00,7.00 Z"/>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(-80.00000,-23.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="164" height="52" rx="7.8" ry="7.8" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(244,184,45)" stroke-opacity="1" fill-opacity="1" style="cursor: default;"/>
        <rect width="160" height="46" rx="6.8999999999999995" ry="6.8999999999999995" x="0" y="0" stroke-width="2px" fill="rgb(80,157,192)" stroke="rgb(57,113,177)" style="cursor: default;"/>
        <text font-family="Verdana" font-size="13.4375" font-style="normal" font-weight="bold" fill="#ffffff" visibility="visible" style="cursor: default;" y="13" x="20">
            <tspan dy="1em" x="20">Central Topic</tspan>
        </text>
        <g preserveAspectRatio="none" focusable="true" width="0" height="26"/>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(161.50000,-100.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="280" height="42" rx="6.3" ry="6.3" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="0" fill-opacity="0"/>
        <line stroke-width="1px" stroke="#495879" stroke-opacity="1" style="cursor: default;" x1="0" y1="36" x2="276" y2="36"/>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="10" x="9">
            <tspan dy="1em" x="9">Topic with undefined shape (uses theme default)</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"/>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"/>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(161.50000,0.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="230" height="42" rx="6.3" ry="6.3" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="0" fill-opacity="0"/>
        <line stroke-width="1px" stroke="#495879" stroke-opacity="1" style="cursor: default;" x1="0" y1="36" x2="226" y2="36"/>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="10" x="9">
            <tspan dy="1em" x="9">Topic with explicit none shape</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"/>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"/>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(161.50000,100.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="180" height="42" rx="6.3" ry="6.3" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="0" fill-opacity="0"/>
        <line stroke-width="1px" stroke="#495879" stroke-opacity="1" style="cursor: default;" x1="0" y1="36" x2="176" y2="36"/>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="10" x="9">
            <tspan dy="1em" x="9">Topic with line shape</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"/>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"/>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(161.50000,200.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="220" height="42" rx="6.3" ry="6.3" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="1" fill-opacity="1"/>
        <rect width="216" height="36" rx="5.4" ry="5.4" x="0" y="0" stroke-width="1px" fill="rgb(252,235,192)" stroke="rgb(0,0,0)" style="cursor: default;"/>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="10" x="9">
            <tspan dy="1em" x="9">Topic with rectangle shape</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"/>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"/>
    </g>
</svg>