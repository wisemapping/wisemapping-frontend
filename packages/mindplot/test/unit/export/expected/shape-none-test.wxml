<map name="shape-none-test" layout="mindmap" version="tango"><topic central="true" text="Central Topic" id="1"><topic position="200,-100" order="0" text="Topic with undefined shape (uses theme default)" id="2"/><topic position="200,0" order="1" text="Topic with explicit none shape" shape="none" id="3"/><topic position="200,100" order="2" text="Topic with line shape" shape="line" id="4"/><topic position="200,200" order="3" text="Topic with rectangle shape" shape="rectangle" id="5"/></topic></map>