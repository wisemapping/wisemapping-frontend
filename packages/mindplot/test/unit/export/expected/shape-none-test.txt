1 Central Topic
	1.1 Topic with undefined shape (uses theme default)
	1.2 Topic with explicit none shape
	1.3 Topic with line shape
	1.4 Topic with rectangle shape
